example/ios/Podfile.lock

```diff
@@ -45,11 +43,11 @@ SPEC CHECKSUMS:
   device_info_plus: 7545d84d8d1b896cb16a4ff98c19f07ec4b298ea
   Flutter: f04841e97a9d0b0a8025694d0796dd46242b2854
   location: 3a2eed4dd2fab25e7b7baf2a9efefe82b512d740
-  MapLibre: 5aa820115d91293a4d09e681e7470f88bd34b672
-  maplibre_gl: 2a0ce99f40b3a133b0e8b40b5ef37e0562165399
-  MapLibreAnnotationExtension: bc7f1041206218d7d24ba0908e3145cd54b86664
+  MapboxMobileEvents: 1176afebbeca86806614a3dc409840bc02a3dd89
```

Comment:
What is the MapboxMobileEvents dependency?

Reply:
https://github.com/maplibre/maplibre-native/blob/5a4d3915232235105efd3689592f5a1f04721081/platform/ios/CHANGELOG.md?plain=1#L179

Reply:
It is a dependency of MapLibre Native.

Reply:
But I was under the impression that this dependency was removed in Maplibre [https://github.com/maplibre/maplibre-native/blob/5a4d3915232235105efd3689592f5a1f04721081/platform/ios/CHANGELOG.md?plain=1#L91](https://github.com/maplibre/maplibre-native/blob/5a4d3915232235105efd3689592f5a1f04721081/platform/ios/CHANGELOG.md?plain=1#L91)

Reply:
My bad for adding it then, I used an old Podfile. I will update it.

Reply:
@JulianBissekkou Please try `5.14.0-pre2` which has the dependency removed.

Reply:
Looks like its gone in `5.14.0-pre3`. We are not able to use `5.14.0-pre2` since it's an debug build